flatpak/org.mamedev.MAME.yaml

```diff
@@ -0,0 +1,71 @@
+id: org.mamedev.MAME
+runtime: org.kde.Platform
+runtime-version: 5.15-21.08
+sdk: org.kde.Sdk
+command: mame
+separate-locales: false
+finish-args:
+  - --device=all
+  - --filesystem=home
```

Comment:
` --persist=.` or better, ` --persist=.mame`, can be used.
Basically the app will see a standard filesystem but on the host only files inside `~./var` will be modified.

It will also not mess with a potential system-wide `mame` installation, each one will have its own files.

Reply:
I see. Still not sure if it's a good idea to pack in a preconfigured mame.ini or not. Wanted to make it overrideable if needed

Reply:
What's sure is giving the whole home directory writable defeats the whole purpose of the Flatpak sandbox.

Make only `.mame` writable at least, and the user can be let to give write access for needed directories.

Reply:
Okay, that can be changed. I would also mention devices=all completely voids any sense of sandbox security whatsoever